brainiak/utils/utils.py

```diff
@@ -767,31 +768,45 @@ def p_from_null(X, two_sided=False):
     X : ndarray with arbitrary number of dimensions
         The last dimension of X should contain the real result in X[..., 0]
         and the null results in X[..., 1:]
+        If max_null_input and min_null_input are provided,
+        X should contain only the real result
 
     two_sided : bool, default:False
         Whether the p value should be one-sided (testing only for being
         above the null) or two-sided (testing for both significantly positive
         and significantly negative values)
 
+    max_null_input
+    min_null_input
```

Comment:
Please follow the parameter doctring format: specify the type on the same line as the parameter name.

To avoid repeating the description, you could say: "See `max_null_input`."

To reference the ISFC module, you should write "see `brainiak.isfc`" (with backticks). This will automatically create a link in the HTML docs, because we use ":py:obj:" as our default Sphinx role:
http://www.sphinx-doc.org/en/stable/domains.html#role-py:obj

Finally, remember sentence capitalization.

Reply:
Thank you, fixed accordingly.